recipes/upstream/digitalocean/rke/main.tf

```diff
@@ -37,13 +38,16 @@ module "rke" {
 }
 
 module "rancher_install" {
-  source                     = "../../../../modules/rancher"
-  dependency                 = module.rke.dependency
-  kubeconfig_file            = module.rke.rke_kubeconfig_filename
-  rancher_hostname           = join(".", ["rancher", module.upstream-cluster.droplets_public_ip[0], "sslip.io"])
-  rancher_replicas           = var.droplet_count
-  rancher_bootstrap_password = var.rancher_password
-  rancher_password           = var.rancher_password
-  rancher_version            = var.rancher_version
-  wait                       = var.wait
+  source                           = "../../../../modules/rancher"
+  dependency                       = module.rke.dependency
+  kubeconfig_file                  = module.rke.rke_kubeconfig_filename
+  rancher_hostname                 = join(".", ["rancher", module.upstream-cluster.https_loadbalancer_ip, "sslip.io"])
```

Comment:
@leodotcloud I'll create a conditional here that will use the node ip as the rancher URL if the user chooses not to create an https loadbalancer resource in the digitalocean module.

Reply:
Completed in https://github.com/rancher/tf-rancher-up/pull/147/commits/31ea706bcaad436b1d3be2c0e063c14f98ff39f3